src/app/material-timepicker/services/time-adapter.spec.ts

```diff
@@ -151,7 +151,7 @@ describe("TimeAdapter", () => {
             expect(isAvailable).toBeTruthy();
         });
 
-        it("should return false since time does not pass filter.", function() {
+        it('should return false since time passes filter.', function() {
```

Comment:
the same test case as above. You might meant `does not pass` ?

Reply:
yes :)